### PR TITLE
[20.6] Add missing shulker box tags

### LINKED ISSUE #894: Convention tag requests
> [!NOTE]
> Any requests for a new tag under the `c:` namespace should also be mirrored to the https://github.com/FabricMC/fabric repository.
> 
> No change to the tags will be entertained that is not brought up for discussion on Fabric's side. 
> 
> Neither will a PR to these tags be merged unless both NeoForge and Fabric have approved the PR with the same tag(s), the same contents, and the same definition of what the tag(s) contains.
> 
> ~ @sciwhiz12

---

Why is there no shulker box tag? 😭 @TelepathicGrunt

### PR DESCRIPTION
**Pull request by Spinoscythe**
Add missing shulker box tags
Fixes #894

### CODE REVIEW

**Comment by Spinoscythe** [comment]
Will be in draft until Fabric PR

**Comment by dhyces** [comment]
Why does fabric need a PR? They already have an item tag for shulker boxes. Also, remember to run datagen.

File: src/main/java/net/neoforged/neoforge/common/Tags.java [hunk 1]
```diff
@@ -184,6 +184,7 @@ public static class Blocks {
         public static final TagKey<Block> SANDSTONE_UNCOLORED_BLOCKS = tag("sandstone/uncolored_blocks");
         public static final TagKey<Block> SANDSTONE_UNCOLORED_SLABS = tag("sandstone/uncolored_slabs");
         public static final TagKey<Block> SANDSTONE_UNCOLORED_STAIRS = tag("sandstone/uncolored_stairs");
+        public static final TagKey<Block> SHULKER_BOXES = tag("shulker_boxes");
```
Comment (dhyces) [inline src/main/java/net/neoforged/neoforge/common/Tags.java]: Block tag is still present

**Comment by TelepathicGrunt** [comment]
You are missing the lang file entries. See the lang datagen file and then datagen the new lang file

**Comment by Spinoscythe** [comment]
Superseded by #942

**Pull request closed without merging**